packages/lesswrong/lib/collections/users/helpers.tsx

```diff
@@ -229,11 +229,11 @@ export const userEmailAddressIsVerified = (user: UsersCurrent|DbUser|null): bool
 };
 
 export const userHasEmailAddress = (user: UsersCurrent|DbUser|null): boolean => {
-  return !!(user?.emails && user.emails.length > 0);
+  return !!user?.email
 }
 
 export function getUserEmail (user: UsersCurrent | DbUser): string | undefined {
```

Comment:
So we have userGetEmail and getUserEmail. I think probably we should convert one of these into the other. (It's also less clear we need this in the world where we actually successfully are only using email)